tools/admin/wskadmin

```diff
@@ -32,7 +32,10 @@ import sys
 import traceback
 import uuid
 import wskprop
-import urllib
+if (sys.version_info > (3, 0)):
```

Comment:
Should this be a comparison of `>=` instead of `>`? Same applies to the below occurrences.